packages/helm-charts/celo-fullnode/rc1-blockscout-archive-nodes.yaml

```diff
@@ -0,0 +1,148 @@
+aws: false
+azure: false
+fullnameOverride: false
```

Comment:
nit: these are the default values

Reply:
Agree, better to just override non-default values.

Reply:
Updated just setting non-default or particularly interesting values here.